app/views/sessions/new.html.haml

```diff
@@ -1,31 +1,13 @@
 %section.sign_in.container
   .row
     .col-sm-10.col-sm-offset-1
-      = form_tag '/sign_in' do 
+      = bootstrap_form_tag url: sign_in_path, class: 'sign_in' do |f| 
         %header
           %h1 Sign in
         %fieldset
-          .form-group
-            = label_tag :email, "Email Address"
-            .row
-              .col-sm-4
-                = email_field_tag :email, nil, class: "form-control"
-          .form-group
-            = label_tag :password, "Password"
-            .row
-              .col-sm-4
-                = password_field_tag :password, nil, class: "form-control"
+          .row
+            = f.email_field :email, wrapper: { class: "col-sm-4"}, label: "Email Address"
```

Comment:
I think you don't need this wrapper, bootstrap_form_for should use layout: :horizontal and set the control_col and label_col there - [check this out](https://github.com/bootstrap-ruby/rails-bootstrap-forms#horizontal-forms).